doc/man/8/radosgw-admin.rst

```diff
@@ -288,13 +487,30 @@ Options
 
 	Temporary url key.
 
+.. option:: --access=<access>
+
+   Set the access permissions for the sub-user.
+   Available access permissions are read, write, readwrite and full.
+
+.. option:: --max-buckets
+
+   Maximum number of buckets for a user.
+
+.. option:: --admin
+
+   Set the admin flag on the user.
+
 .. option:: --system
 
-	Set the system flag on the user.
+   Set the system flag on the user.
 
 .. option:: --bucket=bucket
 
-   Specify the bucket name.
+   Specify the bucket name. It is also used with quota command.
```

Comment:
This is used with many other commands (like metadata, reshard) I would just keep the original text